pkg/agent/openflow/network_policy.go

```diff
@@ -1313,3 +1336,73 @@ func (c *client) ReassignFlowPriorities(updates map[uint16]uint16, table binding
 	}
 	return nil
 }
+
+func parseDropFlow(flow string) (uint32, types.RuleMetric) {
+	// example format:
+	// table=101, n_packets=9, n_bytes=666, priority=200,ip,reg0=0x100000/0x100000,reg3=0x5 actions=drop
+	segs := strings.Split(flow, ",")
+	m := types.RuleMetric{}
+	pkts, _ := strconv.ParseUint(segs[1][strings.Index(segs[1], "=")+1:], 10, 32)
+	m.Packets = pkts
+	bytes, _ := strconv.ParseUint(segs[2][strings.Index(segs[2], "=")+1:], 10, 32)
+	m.Bytes = bytes
+	id, _ := strconv.ParseUint(segs[6][strings.Index(segs[6], "0x")+2:strings.Index(segs[6], " ")], 16, 32)
+	return uint32(id), m
```

Comment:
Sessions is not set? though it equals to Sessions.

Reply:
I thought it is hard to say session if any packet is dropped.
I have assigned sessions with the value of packets.